tests/test_unit/utils.py

```diff
@@ -311,94 +312,32 @@ def create_instance(self, resource_cls: type[T_Object], skip_defaulted_args: boo
         return resource_cls(*positional_arguments, **keyword_arguments)
 
     def create_value(self, type_: Any, var_name: str | None = None) -> Any:
+        import numpy as np
```

Comment:
Replaced this with the implementation from the Cognite-SDK, which accounts for some new type hints (dataclass.InitVar for example)